### PR TITLE
expose emulator debug log

### PR DESCRIPTION
**Pull request by sukantoraymond**
Resolves conflict https://github.com/onflow/flow-cli/pull/535
______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,8 +10,8 @@ require (
 	github.com/onflow/cadence/languageserver v0.24.0
 	github.com/onflow/fcl-dev-wallet v0.4.6
 	github.com/onflow/flow-cli/pkg/flowkit v0.0.0-20220707015502-7b591781ff1b
-	github.com/onflow/flow-emulator v0.33.3
-	github.com/onflow/flow-go-sdk v0.26.4
+	github.com/onflow/flow-emulator v0.33.4
+	github.com/onflow/flow-go-sdk v0.26.5-0.20220629191626-900f9f91bffc
```
Comment (psiemens) [inline go.mod:14]: I think this can just be `github.com/onflow/flow-go-sdk v0.26.5`

`v0.26.5` points to the latest SDK commit on master

**Review by psiemens — COMMENTED** [review]
@sukantoraymond I think the CI errors are occurring because the emulator is using a newer version of Flow Go than the CLI. 

I would delete the `flow-go` indirect dependency in `go.mod` and the run `go mod tidy` to see if that fixes the issue.

File: go.mod (hunk 1 above)
Comment (sukantoraymond) [reply go.mod:14]: This is my go mod
````
module github.com/onflow/flow-cli

go 1.18

require (
	github.com/getsentry/sentry-go v0.13.0
	github.com/go-git/go-git/v5 v5.4.2
	github.com/manifoldco/promptui v0.9.0
	github.com/onflow/cadence v0.24.2-0.20220711220312-9cdb86cad4ef
	github.com/onflow/cadence/languageserver v0.24.1-0.20220711220312-9cdb86cad4ef
	github.com/onflow/fcl-dev-wallet v0.4.6
	github.com/onflow/flow-cli/pkg/flowkit v0.0.0-20220708202053-3b2866146b5f
	github.com/onflow/flow-emulator v0.33.4
	github.com/onflow/flow-go-sdk v0.26.5-0.20220629191626-900f9f91bffc
	github.com/psiemens/sconfig v0.1.0
	github.com/spf13/afero v1.8.2
	github.com/spf13/cobra v1.4.0
)

require (
	cloud.google.com/go v0.100.2 // indirect
	cloud.google.com/go/compute v1.3.0 // indirect
	cloud.google.com/go/iam v0.3.0 // indirect
	cloud.google.com/go/kms v1.4.0 // indirect
	github.com/Microsoft/go-winio v0.4.16 // indirect
	github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7 // indirect
	github.com/a8m/envsubst v1.3.0 // indirect
	github.com/acomagu/bufpipe v1.0.3 // indirect
	github.com/beorn7/perks v1.0.1 // indirect
	github.com/bits-and-blooms/bitset v1.2.2 // indirect
	github.com/btcsuite/btcd v0.22.0-beta // indirect
	github.com/c-bata/go-prompt v0.2.5 // indirect
	github.com/cespare/xxhash v1.1.0 // indirect
	github.com/cespare/xxhash/v2 v2.1.2 // indirect
	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e // indirect
	github.com/davecgh/go-spew v1.1.1 // indirect
	github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f // indirect
	github.com/dgraph-io/badger/v2 v2.2007.4 // indirect
	github.com/dgraph-io/ristretto v0.0.3-0.20200630154024-f66de99634de // indirect
	github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2 // indirect
	github.com/dustin/go-humanize v1.0.0 // indirect
	github.com/ef-ds/deque v1.0.4 // indirect
	github.com/emirpasic/gods v1.12.0 // indirect
	github.com/ethereum/go-ethereum v1.9.13 // indirect
	github.com/fsnotify/fsnotify v1.5.1 // indirect
	github.com/fxamacker/cbor/v2 v2.4.1-0.20220515183430-ad2eae63303f // indirect
	github.com/fxamacker/circlehash v0.3.0 // indirect
	github.com/go-git/gcfg v1.5.0 // indirect
	github.com/go-git/go-billy/v5 v5.3.1 // indirect
	github.com/go-test/deep v1.0.5 // indirect
	github.com/gogo/protobuf v1.3.2 // indirect
	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
	github.com/golang/protobuf v1.5.2 // indirect
	github.com/golang/snappy v0.0.3 // indirect
	github.com/google/go-cmp v0.5.8 // indirect
	github.com/google/uuid v1.3.0 // indirect
	github.com/googleapis/gax-go/v2 v2.1.1 // indirect
	github.com/gorilla/mux v1.8.0 // indirect
	github.com/gorilla/websocket v1.5.0 // indirect
	github.com/gosuri/uilive v0.0.4 // indirect
	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 // indirect
	github.com/hashicorp/golang-lru v0.5.4 // indirect
	github.com/hashicorp/hcl v1.0.0 // indirect
	github.com/imdario/mergo v0.3.12 // indirect
	github.com/improbable-eng/grpc-web v0.12.0 // indirect
	github.com/inconshreveable/mousetrap v1.0.0 // indirect
	github.com/ipfs/go-cid v0.1.0 // indirect
	github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99 // indirect
	github.com/joho/godotenv v1.4.0 // indirect
	github.com/kevinburke/go-bindata v3.22.0+incompatible // indirect
	github.com/kevinburke/ssh_config v0.0.0-20201106050909-4977a11b4351 // indirect
	github.com/klauspost/compress v1.15.1 // indirect
	github.com/klauspost/cpuid/v2 v2.0.14 // indirect
	github.com/libp2p/go-buffer-pool v0.0.2 // indirect
	github.com/libp2p/go-libp2p-core v0.15.1 // indirect
	github.com/libp2p/go-libp2p-tls v0.4.1 // indirect
	github.com/libp2p/go-openssl v0.0.7 // indirect
	github.com/logrusorgru/aurora v2.0.3+incompatible // indirect
	github.com/magiconair/properties v1.8.5 // indirect
	github.com/mattn/go-colorable v0.1.12 // indirect
	github.com/mattn/go-isatty v0.0.14 // indirect
	github.com/mattn/go-runewidth v0.0.13 // indirect
	github.com/mattn/go-tty v0.0.3 // indirect
	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1 // indirect
	github.com/minio/sha256-simd v1.0.0 // indirect
	github.com/mitchellh/go-homedir v1.1.0 // indirect
	github.com/mitchellh/mapstructure v1.4.3 // indirect
	github.com/mr-tron/base58 v1.2.0 // indirect
	github.com/multiformats/go-base32 v0.0.4 // indirect
	github.com/multiformats/go-base36 v0.1.0 // indirect
	github.com/multiformats/go-multiaddr v0.5.0 // indirect
	github.com/multiformats/go-multibase v0.0.3 // indirect
	github.com/multiformats/go-multicodec v0.4.1 // indirect
	github.com/multiformats/go-multihash v0.1.0 // indirect
	github.com/multiformats/go-varint v0.0.6 // indirect
	github.com/onflow/atree v0.4.0 // indirect
	github.com/onflow/cadence-lint v0.0.0-20220711153841-433de417d437 // indirect
	github.com/onflow/flow-core-contracts/lib/go/contracts v0.11.2-0.20220628221832-3d206c1d4790 // indirect
	github.com/onflow/flow-core-contracts/lib/go/templates v0.11.2-0.20220628221832-3d206c1d4790 // indirect
	github.com/onflow/flow-ft/lib/go/contracts v0.5.0 // indirect
	github.com/onflow/flow-go v0.25.13-0.20220712084425-4e330f582843 // indirect
	github.com/onflow/flow-go/crypto v0.24.4 // indirect
	github.com/onflow/flow-nft/lib/go/contracts v0.0.0-20220606143647-47126d1009e2 // indirect
	github.com/onflow/flow/protobuf/go/flow v0.3.1 // indirect
	github.com/onflow/fusd/lib/go/contracts v0.0.0-20211021081023-ae9de8fb2c7e // indirect
	github.com/onflow/sdks v0.4.4 // indirect
	github.com/opentracing/opentracing-go v1.2.0 // indirect
	github.com/pelletier/go-toml v1.9.4 // indirect
	github.com/pkg/errors v0.9.1 // indirect
	github.com/pkg/term v1.1.0 // indirect
	github.com/pmezard/go-difflib v1.0.0 // indirect
	github.com/prometheus/client_golang v1.12.1 // indirect
	github.com/prometheus/client_model v0.2.0 // indirect
	github.com/prometheus/common v0.33.0 // indirect
	github.com/prometheus/procfs v0.7.3 // indirect
	github.com/psiemens/graceland v1.0.0 // indirect
	github.com/rivo/uniseg v0.2.1-0.20211004051800-57c86be7915a // indirect
	github.com/rs/cors v1.8.0 // indirect
	github.com/rs/zerolog v1.26.1 // indirect
	github.com/sergi/go-diff v1.1.0 // indirect
	github.com/sethvargo/go-retry v0.2.3 // indirect
	github.com/sirupsen/logrus v1.8.1 // indirect
	github.com/sourcegraph/jsonrpc2 v0.1.0 // indirect
	github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572 // indirect
	github.com/spaolacci/murmur3 v1.1.0 // indirect
	github.com/spf13/cast v1.4.1 // indirect
	github.com/spf13/jwalterweatherman v1.1.0 // indirect
	github.com/spf13/pflag v1.0.5 // indirect
	github.com/spf13/viper v1.10.1 // indirect
	github.com/stretchr/objx v0.4.0 // indirect
	github.com/stretchr/testify v1.7.5 // indirect
	github.com/subosito/gotenv v1.2.0 // indirect
	github.com/thoas/go-funk v0.9.2 // indirect
	github.com/turbolent/prettier v0.0.0-20220320183459-661cc755135d // indirect
	github.com/uber/jaeger-client-go v2.29.1+incompatible // indirect
	github.com/uber/jaeger-lib v2.4.0+incompatible // indirect
	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
	github.com/vmihailenco/msgpack/v4 v4.3.11 // indirect
	github.com/vmihailenco/tagparser v0.1.1 // indirect
	github.com/x448/float16 v0.8.4 // indirect
	github.com/xanzy/ssh-agent v0.3.0 // indirect
	github.com/zeebo/blake3 v0.2.3 // indirect
	go.opencensus.io v0.23.0 // indirect
	go.uber.org/atomic v1.9.0 // indirect
	golang.org/x/crypto v0.0.0-20220411220226-7b82a4e95df4 // indirect
	golang.org/x/exp v0.0.0-20220706164943-b4a6d9510983 // indirect
	golang.org/x/net v0.0.0-20220418201149-a630d4f3e7a2 // indirect
	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b // indirect
	golang.org/x/sys v0.0.0-20220412211240-33da011f77ad // indirect
	golang.org/x/text v0.3.7 // indirect
	golang.org/x/xerrors v0.0.0-20220411194840-2f41105eb62f // indirect
	gonum.org/v1/gonum v0.11.0 // indirect
	google.golang.org/api v0.70.0 // indirect
	google.golang.org/appengine v1.6.7 // indirect
	google.golang.org/genproto v0.0.0-20220222213610-43724f9ea8cf // indirect
	google.golang.org/grpc v1.45.0 // indirect
	google.golang.org/protobuf v1.28.0 // indirect
	gopkg.in/ini.v1 v1.66.2 // indirect
	gopkg.in/warnings.v0 v0.1.2 // indirect
	gopkg.in/yaml.v2 v2.4.0 // indirect
	gopkg.in/yaml.v3 v3.0.1 // indirect
	lukechampine.com/blake3 v1.1.7 // indirect
)

````

**Comment by sideninja** [comment]
Why was tihs PR created? shouldn't this be resolved on the original PR #535 ?

**Comment by sukantoraymond** [comment]
Created a new PR because I couldn't resolve the go mod conflicts on the original PR.

**Comment by sideninja** [comment]
> Created a new PR because I couldn't resolve the go mod conflicts on the original PR.

Shouldn't that be resolved by the PR authors? or if they need help we should just help them or create a PR on that fork no?

**Comment by bjartek** [comment]
I have no issues with this beeing merged and mine just discarded. Navigating the depenceny hirarchy without internal knowledge is getting quite difficult.

**Comment by bjartek** [comment]
When i run tests here locally i get `gateway/emulator.go:106:15: undefined: emulator.WithLogger` so it looks like the correct version of the emulator is not resolved.

**Comment by bjartek** [comment]
See my pr in #573

**Comment by sideninja** [comment]
@bjartek should this one be closed as well?

**Comment by bjartek** [comment]
I think so, but I do no thave permission to do so.

**Pull request closed without merging**